tests/test_configuration.py

```diff
@@ -799,6 +824,24 @@ def setUpClass(cls):
 	def tearDownClass(cls):
 		sys.path.pop(0)
 
+class TestBar(TestCase):
+	def test_bar(self):
+		import powerline as powerline_module
+		with swap_attributes(config, powerline_module):
+			with get_powerline_raw(config, powerline_module, ext='wm', renderer_module='bar') as powerline:
+				self.assertEqual("%{{l}}{}%{{r}}{}".format(powerline.render(side='left'), powerline.render(side='right')), "SOME_STRING_HERE")
```

Comment:
as I am currently only piping the output to bar, I'll leave it as is (except for the test), should be a small change if there ever is a need to anyway